apps/nextjs-website/src/components/Header.tsx

```diff
@@ -1,10 +1,58 @@
 'use client';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Header as HeaderEC } from '@pagopa/pagopa-editorial-components/dist/components/Header';
 import { HeaderProps } from '@pagopa/pagopa-editorial-components/dist/components/Header/Header';
+import { Stack } from '@mui/material';
+import { usePathname } from 'next/navigation';
 
-const Header: React.FC<HeaderProps> = (headerData) => (
-  <HeaderEC {...headerData} />
-);
+const Header: React.FC<HeaderProps> = (headerData) => {
+  const pathname = usePathname();
+  const [headerDataWithActiveLink, setHeaderDataWithActiveLink] =
+    useState(headerData);
+
+  // eslint-disable-next-line functional/no-return-void
+  useEffect(() => {
```

Comment:
Why do you use `useEffect`?

Reply:
> Why do you use `useEffect`?

Probably it was included to renderize the menu active items. This component, even if is “one” is splitted into 2:

1) the logo name/avatar on the left and ctaButtons on the right of the header
2) the menu items itself that are getted from the Pages that have the possibility to be marked as “active”

I’m sure @simosalsi did useEffect for the “active” element of the items

Reply:
It also works by removing `useEffect` and `useState`, so I was wondering if it's really needed